main.py

```diff
@@ -0,0 +1,27 @@
+import csv
+
+#creating the variables to store the individual values
+temperature = []
+humidity = []
+
+with open("weather_data.csv", "r") as csvfile:
+    reader = csv.reader(csvfile)
+    next(reader)  #to skip the header row
+
+    for row in reader:
+        temperature.append(float(row[1]))
+        humidity.append(int(row[2]))
```

Comment:
Global variables are okay, but if we can, we should try to keep the cope of variables as small as we can. Could we make this into a function that returns the lists?